assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| IPM               | Fully | The project management and environment work flows encourages the creation of integrated product plans but are lacking when it comes to the management of dependencies.            |
+| OPD               | Not at all | Due to it being a project methodology and RUP does not prescribe any method for dealing with organizational process assets.             |
+| OPF               | Partially | RUP is a roject methodology so organizations would need to supplement it with other processes for this area. Although, RUP's environment work flow does have an activity which handles the assessment of the development-organization.            |
+| OT                | Not at all | Identifying training needs of an organization and assessing its effectiveness is not covered in RUP.          |
+| PI                | Fully | RUP addresses product integration through its implementation, test and deployments workflows.            |
+| RD                | Fully | Mainly through its Requirements workflow, but also through others, RUP provides good support for understanding stakeholder needs, produce and refine requirements and manage change requests. |
+| RSKM              | Fully | RUP itself is a risk driven development process. It aims to indentify as many risks as possible and to tackle them in the early phases (elaboration) of the project.            |
+| VAL               | Fully | In RUP validation is covered by a product acceptance plan in the Project Management workflow and through beta testing during Deployment.             |
+| VER               | Fully | Verification is als covered in RUP by means of planning and conducting system and integration tests. Through its *work guideline* Review it also prescribes the practice of peer reviews.        |
+| **Level 4**|
+| OPP               | Not at all | This is not in the scope of RUP.           |
+| QPM               | Not at all | This is not in the scope of RUP.          |
+| **Level 5**        |
+| CAR               | Partially | This can be done in an Iteration Assessment. Howerver, RUP does not prescribe any analysis techniques for it.           |
+| OPM               | Not at all | This is not in the scope of RUP.             |
+
+## HealthCare.gov
+
+| Cmmi Process Area | Covered | Rationale  |
+| ------------------| ------- | ---------  |
+| **Level 2**           |
+| CM                | Unknown | There is no information found on how configuration management was processed in the HealthCare.gov project.           |
+| MA                | Not at all | Obama claimed to not be well notified about the status of the project and an audit by the Inspector General office showed that CMs did not keep adequate track of records [IHBEAT]. This indicates that there were no processes defined to measure aspects and of the project and correctly report them to the relevant parties.             |
```

Comment:
and of the project and correctly report them to the relevant partes. What do you mean with this?